aries-site/src/layouts/content/Example.js

```diff
@@ -88,6 +83,53 @@ const syntax = {
   `,
 };
 
+export const screens = {
+  desktop: 'desktop',
+  mobile: 'mobile',
+  laptop: 'laptop',
+};
+
+const BrowserWrapper = ({ screen, ...rest }) => {
```

Comment:
This results of this are super nice. Great work. 

I do think Example needs to graduate to its own directory and have `Example`, `ExampleControls`, `ResponsiveControls`, `BrowserWrapper` as components which live within.  (The ResponsiveControls modification is perfect)

Probably the `syntax` style definition as well.

Probably the details, code, relevantComponents show/hide block as well.

Reply:
I was thinking the same, that it's time to break this out. I'll do some restructuring in this PR to clean things up.